### PR TITLE
Update to use the new Petabridge Library template

### PR DESCRIPTION
**Pull request by ayoung**
** **WORK IN PROGRESS** **

So far I have only added the Akka.TestKit.NUnit project. But I just wanted to make sure I'm on the right track before I go on to adding the project for NUnit3.

From what I understand, the project that supports NUnit 2.6.4 should only target net45. While the NUnit3 project should target both net45 and netstandard2.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```
Comment (sean-gilliam) [inline LICENSE]: This LICENSE file should stay unless there's a reason for its removal.

File: src/common.props [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<Project>
+  <PropertyGroup>
+    <Copyright>Copyright © 2017 Your Company</Copyright>
+    <Authors>Your Authors</Authors>
+    <VersionPrefix>0.1.0</VersionPrefix>
+    <PackageReleaseNotes>Placeholder</PackageReleaseNotes>
+    <PackageIconUrl></PackageIconUrl>
+    <PackageProjectUrl></PackageProjectUrl>
+    <PackageLicenseUrl></PackageLicenseUrl>
```
Comment (sean-gilliam) [inline src/common.props]: ```<PackageLicenseUrl>``` should use the URL to this repo's LICENSE file.

File: LICENSE (hunk 1 above)
Comment (ayoung) [inline LICENSE]: Looks like it is missing from the Petabridge Templates.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,27 +1,2 @@
-#### 1.2.3 July 10 2017
```
Comment (Aaronontheweb) [inline RELEASE_NOTES.md]: @ayoung would you mind keeping the old release notes and appending to them in this instance?

**Comment by ayoung** [comment]
No prob. I'm incrementally moving things back into this one. Just seemed
easier to do.

**Comment by Aaronontheweb** [comment]
@ayoung could you add the license file back? Otherwise this is good to go.

**Comment by ayoung** [comment]
@Aaronontheweb I'm closing this. Replaced with #25.

**Comment by Aaronontheweb** [comment]
@ayoung 👍

**Pull request closed without merging**